examples/resources/grafana_cloud_observability_aws_account/resource.tf

```diff
@@ -0,0 +1,18 @@
+data "grafana_cloud_stack" "test" {
+  slug = grafana_cloud_stack.test.slug
+}
+
+resource "grafana_cloud_observability_aws_account" "test" {
+  stack_id = data.grafana_cloud_stack.test.id
+  name     = "my-aws-account"
```

Comment:
Thinking out loud things: We are likely to take multiple cloudwatch jobs -> 1 account which makes choosing a name rather meh. We _might_ be able to get the account name from one of the AWS APIs to properly lump them all? 

That's not particularly a now problem but we could also choose to drop the user defined account name all together if we can get their real account name 🤔.

Reply:
For the name, I think I was mainly concerned about having something useful and relatively easy to import on for a resource ID anyways. Since we're switching to a singular role ARN, stack_id + role_arn makes sense for that.